### DISCUSSION TITLE
[Locations] Add admin level 1

### DISCUSSION THREAD

**Discussion by Glazed** [breezy-weather/breezy-weather#983] [Ideas]
### Steps to reproduce

1. Search for a US city name like "Winchester".
2. See four results in the USA, but can't tell which is which.

### Expected behavior

The search results should show an entry for "United States, Indiana" for Winchester, Indiana, and not "United States, Randolph" which is the name of the county in which Winchester resides.

People in the US only know their own county name and maybe a few others around them. When trying to see the weather in a city they will be visiting, they will not know the county name. They will only know the state name. City names in the US are unique per state, showing the state is sufficient and better.

### Actual behavior

Search results show "United States, [county].

![Screenshot_20240419-132403](https://github.com/breezy-weather/breezy-weather/assets/1869438/d5db4d63-81b4-4d02-bff9-c7f9118f7f85)

In addition, trying to search for "Winchester, IN" or "Winchester, Indiana" shows no results.

### Weather source used

OprnMeteo

### Breezy Weather version

5.2.0

### Android version

Android 14

### Device

Pixel 7 Pro

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue.
- [X] I checked that my issue is not covered in the [Frequently Asked Questions](https://github.com/breezy-weather/breezy-weather/blob/main/HELP.md) document.
- [X] I have updated the app to **[version 5.2.0 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)** and I can reproduce the issue in this version.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance), unless my issue only happens with a specific language. I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Currently, we store country, admin level 2, city and district. Adding admin level 1, 3 or 4 requires extending the current database and a nice way to show it in the app without having too much overwhelming information.

> In addition, trying to search for "Winchester, IN" or "Winchester, Indiana" shows no results.

Reproducible:
https://open-meteo.com/en/docs/geocoding-api/#name=Winchester,+Indiana

Check for existing issues on Open-Meteo side or report a new issue here:
https://github.com/open-meteo/geocoding-api/issues

**Comment by papjul** [comment]
Fixed in f3ee681534c6acaeb46f4f2070d3576c2a4304c3
